data/migrations/dist/20220331034313_children.dev.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+exports.up = function (knex) {
+  return knex.schema.createTable('children', function (table) {
+    table.string('childId').notNullable().unique().primary();
```

Comment:
What is the value of this "string" primary key? Should it just be an `increments('id').primary()`?